packages/client/src/styles/globals.css

```
@@ -1233,13 +1291,13 @@ input[type="range"]:disabled {
-  --range-track-color: var(--marinara-chat-chrome-panel-divider);
-  --range-fill-color: var(--marinara-chat-chrome-button-text-active);
-  --range-thumb-color: var(--marinara-chat-chrome-button-text-active);
+  --range-track-color: color-mix(in srgb, #1db954 26%, transparent);
+  --range-fill-color: #1db954;
+  --range-thumb-color: #1db954;
   --range-thumb-size: 0.6875rem;
   --range-track-height: 0.25rem;
-  --range-thumb-shadow: 0 0 0 0.125rem var(--marinara-chat-chrome-panel-bg);
-  accent-color: var(--marinara-chat-chrome-button-text-active);
+  --range-thumb-shadow: 0 0 0 0.125rem #191414;
+  accent-color: #1db954;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)